packages/client-http/src/ApplyManager.ts

```diff
@@ -6,9 +6,13 @@ export interface ApplyManagerOptions {
   client: ConfidenceClient;
 }
 
+type TokenRecord = {
+  pendingApply: AppliedFlag[];
+  seenFlags: Set<string>;
+};
+
 export class ApplyManager {
-  private resolveTokenPending: Map<string, AppliedFlag[]> = new Map();
-  private resolveTokenSeen: Map<string, Set<string>> = new Map();
+  private readonly tokenRecords: Map<string, TokenRecord> = new Map();
```

Comment:
Since both these maps were keyed by resolveToken I baked them into one using the new TokenRecord type.